src/components/forms/Input/Input.tsx

```diff
@@ -71,15 +70,4 @@ const Input = styled.input.attrs<InputProps>(({ isDisabled }) => ({
   }
 `;
 
-Input.propTypes = {
-  isInvalid: PropTypes.bool,
-  isDisabled: PropTypes.bool,
-};
-
-Input.defaultProps = {
-  isInvalid: false,
-  isDisabled: false,
-  type: 'text',
-};
```

Comment:
@silvioprog `defaultProps` should stay for now, we have different ticket for it and there is migration needed